js/h5p.js

```diff
@@ -1637,7 +1637,7 @@ H5P.shuffleArray = function (array) {
  *   Reported time consumption/usage
  */
 H5P.setFinished = function (contentId, score, maxScore, time) {
-  if (score && H5PIntegration.postUserStatistics === true) {
+  if (typeof score === 'number' && H5PIntegration.postUserStatistics === true) {
```

Comment:
Note that `typeof score` won't equal `number` if someone has parsed it using e.g. `new Number(123)`. Must use `instanceof` as well to support all use-cases.